portfolio/src/main/webapp/index.html

```diff
@@ -41,6 +41,22 @@
       <!-- Console -->
       <div id="console"></div>
 
+      <!--projects -->
+      <div id="projects_inv" class= "container">
+        <div class=" row row-cols-2">
+          <div class= "col-5">
+            <img src='./images/space_invaders.gif' class='project_img mx-auto d-block' alt='invaders gif'>            
+          </div>
+          <div class= "col-7">
+            <p>Built a <span class='emphasis'>retro shooter game</span> in Python under the supervision of Prof. Walker White.
+              The game is designed to vaguely resemble the popular space invaders arcade game from 1978. I used 8-bit images to connote the aliens and the spaceship
+              . The game employed a round system with each round having increased alien speed. The player controls the ship and tries to eliminate all the aliens before his lives run out.
+              I designed extra life and extra laser bolt powerups. This project relied heavily on <span class='emphasis'>the concept of inheritance</span>.
+            </p>
+          </div>
+        </div>
+      </div>
```

Comment:
format changes in later commits, this setup worked at first but needed to be tweaked